### DISCUSSION TITLE
Problem of linking additional device

### DISCUSSION THREAD

**Discussion by LinosWu** [getlantern/lantern-forum-cn#695] [Q&A]
Dear Madam/Sir:
     I have problems here. I can't link additional device to my Lantern Pro account. Is it something to do with the system? My account is using in Windows but the other device is Mac. I thought that it was about the version of the app, so I uninstalled my lantern and reinstalled it from the Lantern website. But nothing changed. My email still doesn't work. I entered the linking pin but it said my device has been successfully added. But after we restarted the app, we still can't see the additional device. Looking forward to your reply! Thank you !
<img width="259" alt="微信图片_20240705175229" src="https://github.com/getlantern/lantern-forum-cn/assets/174789556/7a894a92-fdc3-481f-90d1-86b46539a637">

**Comment by LinosWu** [comment]
I've already sent my report in phone app and the ticket number is 74279228.

**Reply by MaxMKMax** [reply]
Okay thank you so much. I will help you to raise a ticket for the tech team to solve it.

**Reply by LinosWu** [reply]
Thank you! Looking forward to it!

**Reply by LinosWu** [reply]
Hello Mike, when can I get my problem solved? If it can't be solved, I'm gonna request refund. Is there really a tech team?
